patches/server/0049-Linear-region-file-format.patch

```diff
@@ -351,9 +350,7 @@ index 0000000000000000000000000000000000000000..7f95e17544b762ad4674315e53fcff8e
 +        return EnumConfigCategory.MISC.getBaseKeyName() + ".region-format-settings";
 +    }
 +
-+    @DoNotLoad
 +    private static final Logger logger = LogUtils.getLogger();
-+    @DoNotLoad
```

Comment:
Do not remove these two annotations, they are necessary for the future config reload udpate